test/test_grid.py

```diff
@@ -952,7 +952,38 @@ def test_populate_bounds_GCA_mix(self):
         face_bounds = bounds_xarray.values
         nt.assert_allclose(grid.bounds.values, expected_bounds, atol=ERROR_TOLERANCE)
 
-    def test_opti_bounds(self):
-        import uxarray
-        uxgrid = ux.open_grid(gridfile_CSne8)
-        bounds = uxgrid.bounds
+    def test_populate_bounds_MPAS(self):
+        xrds = xr.open_dataset(self.gridfile_mpas)
+        uxgrid = ux.Grid.from_dataset(xrds, use_dual=True)
```

Comment:
is something missing here? 
why does it `pass` at the end?

which grid has non normalized cart coords for testing? it would good to document it.

Reply:
The `pass` is most likely left over from the initial writing of the test case.

I believe this was from a merge conflict that I was resolving. Will fix this.